app/controllers/users_controller.rb

```diff
@@ -7,10 +7,21 @@ def new
     def create
         user = user_params
         user[:email] = user[:email].downcase
-        new_user = User.create!(user)
-        session[:user_id] = new_user.id
-        flash[:success] = "Welcome, #{new_user.email}!"
-        redirect_to "/users/#{new_user.id}"
+        new_user = User.new(user)
+
+
+        if new_user.save
+          # session[:user_id] = new_user.id
```

Comment:
We'll need this line in or the user will have to log-in after saving. If you want to update it to the api build you can use  cookies.encrypted[:user_id] = new_user.id